components/Blog/Card/Title.tsx

```diff
@@ -0,0 +1,25 @@
+import Image from 'next/image';
+import Link, { LinkProps } from 'next/link';
+
+interface Props {
+   link: LinkProps;
+   children: React.ReactNode;
+}
+
+const Title = ({ children, link }: Props) => {
+   return (
+      <div className="flex items-center justify-between">
+         <h3 className="text-2xl font-semibold text-gray-900">{children}</h3>
```

Comment:
what happens if I click on this title? is not it better to routing on text too like an anchor tag?